content/blog/glc-scim-api-integration-with-okta-scim-adapter-1.md

```diff
@@ -0,0 +1,72 @@
+---
+title: HPE GreenLake Flex Solutions SCIM API Integration with Okta SCIM Adapter
+date: 2024-05-13T20:55:22.189Z
+author: Namik Rujnic - Cloud Operations
+authorimage: /img/Avatar1.svg
+disable: false
+tags:
```

Comment:
Tags must be added "Individually". Click "Add tag" in the CMS to add tag one by one.